classes/class-p4-search.php

```diff
@@ -168,69 +170,159 @@ public function load( $search_query, $selected_sort = self::DEFAULT_SORT, $filte
 		 * Gets the paged posts that belong to the next page/load and are to be used with the twig template.
 		 */
 		public function get_paged_posts() {
-			// If this is an ajax call.
 			if ( wp_doing_ajax() ) {
```

Comment:
If you flip the logic so that it's `if ( !wp_doing_ajax )` then you can return if that's the case and avoid the rest of the function being wrapped in else.

Reply:
It's an easy refactor, but sometimes it can make your PR change a lot of lines because of the indent change. But in this case it's definitely worth it as all the lines in the method are changing anyway.